.ci/macos/dependencies/gource.sh

```
@@ -0,0 +1,56 @@
+# Dependencies for Gource
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)